source/features/clean-header-search-field.tsx

```diff
@@ -5,7 +5,14 @@ import select from 'select-dom';
 import features from '../feature-manager';
 
 async function init(): Promise<void> {
-	const headerSearchButton = await elementReady('button.header-search-button');
+	const headerSearchButton = await elementReady('button.header-search-button, input.header-search-input');
+	if (headerSearchButton instanceof HTMLInputElement) {
+		// Previous version #6310
+		// TODO: Remove in mid 2023
```

Comment:
Maybe put a real date so it shows up as a warning

Reply:
I know we've been doing that, but what does that change in practice? We don't want CI to start failing and force us to immediately take action 🤔

Reply:
If I am not mistaken we set it to warn

Reply:
We did, but due to: 

- https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1540

I think `expiring-todo-comments` only makes sense:

- if it never breaks (on a date) and forces anyone to act immediately
- if someone is actively looking for and fixing TODO comments 
- if there are no other TODOs that are _also_ waiting for a fix anyway, for months

Essentially I think we can do away with them. Moving this discussion upstream in https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1541